src/docfx/lib/markdown/MarkdownVisitContext.cs

```diff
@@ -8,6 +8,8 @@ namespace Microsoft.Docs.Build
 {
     internal class MarkdownVisitContext
     {
+        public Stack<MonikerList> ZoneMonikerStack { get; set; }
```

Comment:
nit: consider make these 2 stacks private (and pass it through constructor)? (so `Visit` callback cannot see see / change them)